src/ui-shell/header/hamburger.component.ts

```diff
@@ -10,15 +14,45 @@ import { I18n } from "../../i18n/i18n.module";
 	selector: "ibm-hamburger",
 	template: `
 		<button
+			(click)="doClick($event)"
+			[ngClass]="{'bx--header__action--active': active}"
 			class="bx--header__menu-trigger bx--header__action"
 			[attr.aria-label]="i18n.get('UI_SHELL.HEADER.MENU') | async"
 			[attr.title]="i18n.get('UI_SHELL.HEADER.MENU') | async">
-			<svg aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+
```

Comment:
Are there icons from `@carbon/icons-angular` we can pull for these? We can now use the `<svg ibmIconIconName></svg>` style to reference them (since I _think_ that was part of the reason we went with raw SVG here)